.github/workflows/template/template.yml.j2

```diff
@@ -0,0 +1,537 @@
+name: Build git-annex on {{osname}}
```

Comment:
name the file `build-{{osname}}.yml.j2` and use the same dictionary to "format"?

Reply:
The script already determines the workflow names by templating `build-{{ostype}}.yml`.  I don't see an advantage from naming the template to match that.

Reply:
To avoid looking for the right template if we get multiple in the future.

Reply:
And if patches change

Reply:
Done.

Reply:
I do not see any related changes, which commit?

Reply:
This commit: https://github.com/datalad/git-annex/pull/22/commits/2a07b990b5858eac1a884a5ee378009210876189

Reply:
ah, sorry, I conflated two issues in this single thread.  I was wondering about 

> And if patches change

i.e. adding `patches/*` into the list of paths for workflows to react on

Reply:
The workflows now trigger on PRs that change `patches/*.patch`.